src/main/java/org/ideoholic/curium/model/documents/service/DocumentService.java

```diff
@@ -131,37 +135,33 @@ public String generateTransferCertificate() {
-				 request.setAttribute("leavingReason", leavingReason);
-					request.setAttribute("dateinword", dateinword);
-					request.setAttribute("leavingReason", leavingReason);
-					request.setAttribute("bookno", bookno);
-					request.setAttribute("tcno", tcno);
-					request.setAttribute("caste", caste);
-					request.setAttribute("classinword", classinword);
-					request.setAttribute("lastexam", lastexam);
-					request.setAttribute("failpass", failpass);
-					request.setAttribute("firstsubject", firstsubject);
-					request.setAttribute("secondsubject", secondsubject);
-					request.setAttribute("thirdsubject", thirdsubject);
-					request.setAttribute("Fourthsubject", Fourthsubject);
-					request.setAttribute("Fifthsubject", Fifthsubject);
-					request.setAttribute("sixthsubject", sixthsubject);
-					request.setAttribute("pinfig", pinfig);
-					request.setAttribute("pinword", pinword);
-					request.setAttribute("dues", dues);
-					request.setAttribute("concession",concession);
-					request.setAttribute("workingdays", workingdays);
-					request.setAttribute("present", present);
-					request.setAttribute("ncc", ncc);
-					request.setAttribute("game", game);
-					request.setAttribute("conduct", conduct);
-					request.setAttribute("datecert", datecert);
-					request.setAttribute("Remarks", Remarks);
-					
-				 request.setAttribute("dateinword", dateinword);
-				 request.setAttribute("studentdetails", parents);
-				 request.setAttribute("tcdetails", tc);
-				 return "studentexists";
+				 transferCertificateResponseDto.setReason(leavingReason); 
+				 transferCertificateResponseDto.setBookNo(bookno);
+				 transferCertificateResponseDto.setTcNo(tcno);
+				 transferCertificateResponseDto.setCaste(caste);
+				 transferCertificateResponseDto.setClassInWord(classinword);	
+				 transferCertificateResponseDto.setLastExam(lastexam);	
+				 transferCertificateResponseDto.setFailPass(failpass);	
+				 transferCertificateResponseDto.setFirstSubject(firstsubject);
+				 transferCertificateResponseDto.setSecondSubject(secondsubject);	
+				 transferCertificateResponseDto.setThirdSubject(thirdsubject);
+				 transferCertificateResponseDto.setFourthSubject(Fourthsubject);
+				 transferCertificateResponseDto.setFifthSubject(Fifthsubject);
+				 transferCertificateResponseDto.setSixthSubject(sixthsubject);
+				 transferCertificateResponseDto.setPinFig(pinfig);
+				 transferCertificateResponseDto.setPinWord(pinword);	
+				 transferCertificateResponseDto.setDues(dues);
+				 transferCertificateResponseDto.setConcession(concession);
+				 transferCertificateResponseDto.setWorkingDays(workingdays);
+				 transferCertificateResponseDto.setPresent(present);
+				 transferCertificateResponseDto.setNcc(ncc);
+				 transferCertificateResponseDto.setGame(game);
+				 transferCertificateResponseDto.setConduct(conduct);
+				 transferCertificateResponseDto.setDateCert(datecert);
+				 transferCertificateResponseDto.setRemarks(Remarks);
```

Comment:
Also need to add:
`transferCertificateResponseDto.setDateInWord(dateinword);`

Reply:
done